packages/react-components/react-table/src/components/DataGrid/DataGrid.test.tsx

```diff
@@ -34,6 +39,16 @@ describe('DataGrid', () => {
       items: testItems,
       columns: testColumns,
     },
+    testOptions: {
+      'consistent-callback-args': {
+        ignoreProps: ['onColumnResize'],
+      },
+      'make-styles-overrides-win': {
+        callCount: 2,
```

Comment:
the new `useTableColumnSizing_unstable` causes the DataGrid to re-render, which is by design (needs to render first, measure and re-render)

Reply:
This will be addressed in a future ticket @ling1726 will create, where the table plugins will not return hooks (and therefore can be conditionally used)

Reply:
tracked in #26740